src/main/java/nextstep/ladder/domain/ladder/Line.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.ladder.domain.ladder;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Collectors;
+
+public class Line {
+    List<Boolean> line = new ArrayList<>();
```

Comment:
```suggestion
    private final List<Boolean> line = new ArrayList<>();
```
인스턴스 변수는 가능하면 private final로 구현해 보는 연습을 한다.